synfig-core/src/synfig/valuenode_registry.cpp

```diff
@@ -52,23 +52,13 @@ String
 ValueNodeRegistry::localize_name(const String &local_name)
 	{ return _(local_name.c_str()); }
 
-void
+bool
```

Comment:
why `bool` if it always will return true?

Reply:
To call the auto-registration function "automatically", we create a static variable. In this case, we need some sort of return value.

So all the "magic" of auto-registration is to make something like this:
```
static bool result = register_me("class_name");
```
You can't do this on void function.